{% load staticfiles %}
<html>

<head>
	<title>RPGeNet v2.0 - Retinitis Pigmentosa Gene Network</title>
	<meta name="description" content="RPGeNet v2.0 - Retinitis Pigmentosa Gene Network: interacting with the network of genes related to retinitis pigmentosa genes.">
	<meta name="keywords" content="RPGeNet,retinitis pigmentosa,RP,interaction network,retinal disease,driver genes">
	<link rel="stylesheet" href="{% static 'css/style.css' %}">
	<link rel="stylesheet" href="{% static 'css/bootstrap-min.css' %}">
</head>


<body>

	<div id="title-div">
		<div id="title-container" class="row">
			<div class="col-sm-2" id="app-logo-div">
				<a href="/RPGeNet"><img id="app-logo" src="{% static 'Images/logo.png' %}"></a>
			</div>
			<div class="col-sm-10" id="title-text"> RPGeNet <span class="version">v2.0</span> </div>
		</div>


	</div>

			<!-- HELP DIALOGS -->
		<div id="inputgenes">
			<h2>INPUT GENES</h2>
			<hr>
			<p>Here you can input the <em>retinitis pigmentosa</em> related driver gene (RP/LCA genes), for which you want to see the main interactors. <br />You can search for any other gene, yet remember that this is a network built around RP/LCA causative genes and little info can be found for genes outside their scope.<br /><br />
				You can make queries for <b>single (i)</b> or <b>multiple genes (ii)</b>:<br />
				&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b> i)</b> Gene1<br />
				&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;      <b>ii)</b> gene1,Gene2, GeNe3,...<br />
				&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; (case-insensitive, with or without spaces after the comma)<br /><br><br>
				See also tutorial <a href="{% url tutorial %}" target="_blank">page</a>.
			</p>
		</div>

		<div id="inputgenes2">
			<h2>INPUT GENES</h2>
			<hr>
			<p>Here you can input the <em>retinitis pigmentosa</em> related driver gene (RP/LCA genes), for which you want to see the main interactors. <br />You can search for any other gene, yet remember that this is a network built around RP/LCA causative genes and little info can be found for genes outside their scope.<br /><br />
				You can make queries for <b>single (i)</b> or <b>multiple genes (ii)</b>:<br />
				&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b> i)</b> Gene1<br />
				&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;      <b>ii)</b> gene1,Gene2, GeNe3,...<br />
				&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; (case-insensitive, with or without spaces after the comma)<br /><br><br>
				See also tutorial <a href="{% url tutorial %}" target="_blank">page</a>.
			</p>
		</div>


		<div id="genedistance">
			<h2>NEIGHBOURHOOD GENES</h2>
			<hr>
			<p>
				Here you can specify how many "neighbours" your query gene/s in the network shoud have. 
				This just means that starting from your query gene/s you can just "travel" to another node with a distance equal to the value that you specified.
			</p>
			<p>
				For example, when searching for <i>CEP290</i> and the "neighbours" is set to 1 (this is the default value), this set of genes can be obtained: <i>OFD1</i>, <i>RPGR</i>, <i>ANAPC2</i>, <i>ATF4</i>, <i>IQCB1</i>. This means that from <i>CEP290</i> one can only travel to nodes (say here genes) at distance 1. If "neighbours" is set to 2,further genes are retrieved apart from the previous ones, and now one can travel a distance of 2 to other genes.
				<br><br>
				See also tutorial page.
			</p>
		</div>

		<div id="intlvl">
			<h2>INTERACTION LEVEL</h2>
			<p>Here you can specify what interaction file to be used.<br /><br />There are <b>4</b> interaction files describing the level of interaction: level 1, level 2, level 3 and level 4 (we choose the interaction file just by putting the number of the level, i.e 1, 2, 3 or 4). Levels above 2 may take too much to render as many nodes can be included in the drawn subnetwork. <br /><br />
				There is also a level 0 (based on skeleton network), which represents the interactions found from network shortest paths between all pairs of RP/LCA driver genes, used as "seed" to build all the other levels.<br /><br />
			</p>
			See also tutorial <a href="{% url tutorial %}" target="_blank">page</a>.
		</div>

		<div id="intlvl2">
			<h2>INTERACTION LEVEL</h2>
			<p>Here you can specify what interaction file to be used.<br /><br />There are <b>4</b> interaction files describing the level of interaction: level 1, level 2, level 3 and level 4 (we choose the interaction file just by putting the number of the level, i.e 1, 2, 3 or 4). Levels above 2 may take too much to render as many nodes can be included in the drawn subnetwork. <br /><br />
				There is also a level 0 (based on skeleton network), which represents the interactions found from network shortest paths between all pairs of RP/LCA driver genes, used as "seed" to build all the other levels.<br /><br />
			</p>
			See also tutorial <a href="{% url tutorial %}" target="_blank">page</a>.
		</div>

		<div id="genexp">
			<h1>GENE EXPRESSION</h1>
			<p>
				Here you can specify how to visualize the color gradients related to gene-expression levels used to color-fill the node shapes.<br /><br />
				You can choose between <b>Relative</b> and <b>Absolute</b>. <br /><br />"Relative" gene-expression is based in the expression fold-change in retina versus the average expression on all tissues for each gene. This fold-change goes from -3 to +3, ranging from dark yellow (-3, more expressed in other tissues than in retina), white (0, no changes), to dark blue (+3, more expressed in retina than in other tissues).<br /><br />"Absolute" gene-expression considers levels on retina only, and range from null expression in retina (dark yellow) to the highest expression level in retina (darkblue).<br /><br />
			</p>
			See also tutorial <a href="{% url tutorial %}" target="_blank">page</a>.
		</div>

		<div id="genexp2">
			<h1>GENE EXPRESSION</h1>
			<p>
				Here you can specify how to visualize the color gradients related to gene-expression levels used to color-fill the node shapes.<br /><br />
				You can choose between <b>Relative</b> and <b>Absolute</b>. <br /><br />"Relative" gene-expression is based in the expression fold-change in retina versus the average expression on all tissues for each gene. This fold-change goes from -3 to +3, ranging from dark yellow (-3, more expressed in other tissues than in retina), white (0, no changes), to dark blue (+3, more expressed in retina than in other tissues).<br /><br />"Absolute" gene-expression considers levels on retina only, and range from null expression in retina (dark yellow) to the highest expression level in retina (darkblue).<br /><br />
			</p>
			See also tutorial <a href="{% url tutorial %}" target="_blank">page</a>.
		</div>


	<div class="content">
		<div class=" app-panel-container">
			<div class="row">
				<div class="col-sm-6 app-panel">
					<span class="label label-danger app-label">About</span><br><br>
					<p>
						This framework represents the retinitis pigmentosa gene interaction network, built upon 110 RP/LCA driver genes. It combines data from the iHOP, BioGRID, and STRING repositories, with additional information from GeneOntology, GeneCards, OMIM, UniProt, RefSeq, and UCSC.
					</p>
					<p>
						Further details are available on the RPGeNet paper:
					</p>
					<p>
						"Distilling a Visual Network of Retinitis Pigmentosa Gene-Protein Interactions to Uncover New Disease Candidates"
					</p>
					<p>
						<a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0135307" target="_blank">Boloc et al, PLoS ONE, 2015, 10(8): e0135307</a>
					</p>
				</div>
				<div class="app-spacing">&nbsp</div>
				<div class="col-sm-6 app-panel">
					<span class="label label-danger app-label">How to use RPGeNet</span><br><br>
					<p>
						Input query genes separated by commas, semicolons and/or spaces (case insensitive) in order to visualize a focused network on the query then click on "Explore Network", or choose one of the 110 RP/LCA driver genes from the dropdown list to visualize the paths that this gene shares with other RP/LCA genes and click "Get Path to Genes". Here you have some examples of query genes:
					</p>
					<table class="app-table">

						<tr class="spaceUnder"> <th> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </th> <th align="right">Single Genes:  </th> <td align="left">CERKL</td> </tr>
						<tr class="spaceUnder"> <th> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </th> <th align="right">Multiple Genes:</th> <td align="left">EYS, RHO, cerkl,RP1</td> </tr>
					</table>
					<a href="{% url tutorial %}">
						<span class="label label-danger app-label-right">Go to Tutorial</span><br><br>
					</a>
				</div>

			</div>
		</div>



		<div id="forms-container-container">
			<div id="forms-container">


				<form id="f1" action="{% url gene_explorer %}">
					<div class="col-md-3 form-group">
						<label for="f1-genename">Gene/s <span id="inputgenes-btn" class="glyphicon glyphicon-question-sign question-badge"></span></label>
						<input type="text" name="gene" id="f1-genename" class="form-control" placeholder="CERKL"/>
					</div>
					<div class="col-md-2 form-group">
						<label for="f1-distance">Gene distance <span id="genedistance-btn" class="glyphicon glyphicon-question-sign question-badge"></span></label>
						<input type="number" name="dist" id="f1-distance" value="1" min="1" max="5" step="1" />
					</div>
					<div class="col-md-2 form-group">
						<label for="f1-lvl">Interaction level <span id="intlvl-btn" class="glyphicon glyphicon-question-sign question-badge"></span></label>
						<input type="number" name="lvl" id="f1-lvl" value="1" min="1" max="5" step="1" />
					</div>

					<div class="col-md-2 form-group align-center">
						<label for="f1-expression">Expression <span id="genexp-btn" class="glyphicon glyphicon-question-sign question-badge"></span></label>
						<select name="exp" class="form-control" id="f1-expression">
							<option>Experiment-1</option>
							<option>Experiment-2</option>
							<option>Experiment-3</option>
							<option>Experiment-4</option>
							<option>Experiment-5</option>
						</select>
					</div>

					<div class="col-md-3 form-group align-center">
						<label for="created_at_lt">&nbsp;</label>
						<input type="submit" value="Explore Network" class="btn btn-f1 form-control" />
					</div>
				</form>

				<div class="col-md-12"></div>
				<form id="f2" action="{% url pathway_explorer %}">

					<div class="col-md-3 form-group">
						<label for="f2-genename">Gene/s <span id="inputgenes-btn2" class="glyphicon glyphicon-question-sign question-badge"></span></label>
						<input type="text" name="gene" id="f2-genename" class="form-control" placeholder="RHO"/>
					</div>
					<div class="col-md-2 form-group">
						<label for="f2-lvl">Interaction level <span id="intlvl-btn2" class="glyphicon glyphicon-question-sign question-badge"></span><br></label>
						<input type="number" name="lvl" id="f2-lvl" value="1" min="1" max="5" step="1" />
					</div>

					<div class="col-md-2 form-group align-center">
						<label for="f2-expression">Expression <span id="genexp-btn2" class="glyphicon glyphicon-question-sign question-badge"></span></label>
						<select name="exp" class="form-control" id="f2-expression">
							<option>Experiment-1</option>
							<option>Experiment-2</option>
							<option>Experiment-3</option>
							<option>Experiment-4</option>
							<option>Experiment-5</option>
						</select>
					</div>

					<div class="col-md-3 form-group align-center">
						<label for="created_at_lt">&nbsp;</label>
						<input type="submit" value="Get Path Genes" class="btn btn-f2 form-control" />
					</div>
				</form>


			</div>
		</div>


	</div>

	<div id="footer">
		<div class="footer-left">
			<div class="cgl-logo-footer-container">
				<a target="_blank" href="https://compgen.bio.ub.edu">
					<img class="cgl-logo-footer" src="{% static 'Images/compgen-logo.png' %}">
				</a>
			</div>
			<div class="cgl-links-footer">
				<a target="_blank" href="https://compgen.bio.ub.edu">Home</a> · 
				<a target="_blank" href="https://compgen.bio.ub.edu/Research">Research</a> · 
				<a target="_blank" href="https://compgen.bio.ub.edu/Acknowledgements">Acknowledgements</a> · 
				<a target="_blank" href="https://compgen.bio.ub.edu/contact">Contact</a> 
				<p class="copyright">Computational Genomics Lab © 2015-2018</p>
			</div>
		</div>
		<div class="footer-right">
			<div class="logo-container row">
				<div class="col-sm-4">
					<a target="_blank" href="http://www.ub.edu/web/ub/en/index.html?">
						<img class="ub-logo-footer" src="{% static 'Images/ub-logo.png' %}">
					</a>
				</div>
				<div class="col-sm-4">
					<a target="_blank" href="http://www.ub.edu/ibub/en/index.html">
						<img class="ibub-logo-footer" src="{% static 'Images/ibub-logo.png' %}">
					</a>
				</div>
				<div class="col-sm-4">			
					<a target="_blank"  href="http://www.ciberer.es/en">
						<img class="ciberer-logo-footer" src="{% static 'Images/ciberer-logo.png' %}">
					</a>
				</div>
			</div>
		</div>
	</div>
	<script src="{% static 'js/jquery-min.js' %}"></script>
	<script src="{% static 'js/bootstrap-min.js' %}"></script>
	<script src="{% static 'js/stoopidoverlay.js' %}"></script>
	<script>
		var options = {
        container: 'inputgenes',
        openContainer: 'inputgenes-btn',
        scroll: false,
        align: 'center',
        width: '35%',
        background: "white", 
        closeBackground: "white",
        padding: "2%"
      }
      // Gene Overlay
      var overlayGenes      = new StoopidOverlay(options);
      options.container     = 'inputgenes2';
      options.openContainer = 'inputgenes-btn2';
      var overlayDist       = new StoopidOverlay(options);
      // Distance Overlay
      options.container     = 'genedistance';
      options.openContainer = 'genedistance-btn';
      var overlayDist       = new StoopidOverlay(options);
	  // Intlvl Overlay
      options.container     = 'intlvl';
      options.openContainer = 'intlvl-btn';
      var overlayDist       = new StoopidOverlay(options);
      options.container     = 'intlvl2';
	  options.openContainer = 'intlvl-btn2';
	  var overlayDist2      = new StoopidOverlay(options);

	  // Genexp Overlay
	  // Intlvl Overlay
      options.container     = 'genexp';
      options.openContainer = 'genexp-btn';
      var overlayGexp       = new StoopidOverlay(options);
      options.container     = 'genexp2';
	  options.openContainer = 'genexp-btn2';
	  var overlayGexp2      = new StoopidOverlay(options);
      //options.container = 'genedistance';
      //options.openContainer = 'genedistance-btn';
      //var myOverlay2 = new StoopidOverlay(options);
  	</script>
</body>


</html>